sandbox/src/LargePayload.js

```diff
@@ -19,20 +19,39 @@ export default function LargePayload() {
       <h2>Large Payloads</h2>
       <p>This page tests send really large payloads to the edge.</p>
       <p>
-        All those requests should not use beacon calls, and should not fail.
```

Comment:
This wasn't accurate. The command options include `documentUnloading: true`, which means Alloy _does_ use `sendBeacon`. Depending on the browser, some payload will fail to be sent using `sendBeacon` and Alloy will fall back to `fetch`.

Reply:
Wouldn't this be configured without documentUnloading to avoid sendBeacon limit then?

Reply:
I'm not sure I understand your question, but this page is to test what happens when `sendBeacon` _is_ used, so I believe the configuration (`documentUnloading: true`) is correct, but this description we had previously didn't make sense and didn't correlate with our configuration.